crates/editor/src/editor.rs

```diff
@@ -6801,10 +6801,10 @@ impl Editor {
                         break 'expand_downwards;
                     }
                 }
-            }
 
-            if !should_rewrap {
-                continue;
+                if !should_rewrap {
```

Comment:
This change seems a bit suspect.

Wouldn't this mean that for non-empty selections we would always try to wrap them?